### ISSUE TITLE
User spawned forever in all user 

### ISSUE THREAD

**Issue by kirys** [nextcloud/server#21374]
I have configured a nextcloud  server for my home network with ldap auth and somehow in everyone the only currently active user duplicated forever, while in the group is corectly show only once.
![bugnextcloud](https://user-images.githubusercontent.com/20193924/84376428-6c3ab700-abe1-11ea-8abf-c8edbf8c9bc0.png)

```
+-------------------------------+-------------------------------------------------------------------------------------------------------------------+
| Configuration                 | s01                                                                                                               |
+-------------------------------+-------------------------------------------------------------------------------------------------------------------+
| hasMemberOfFilterSupport      | 1                                                                                                                 |
| homeFolderNamingRule          |                                                                                                                   |
| lastJpegPhotoLookup           | 0                                                                                                                 |
| ldapAgentName                 | cn=**********                                                                                                     |
| ldapAgentPassword             | ***                                                                                                               |
| ldapAttributesForGroupSearch  |                                                                                                                   |
| ldapAttributesForUserSearch   |                                                                                                                   |
| ldapBackupHost                |                                                                                                                   |
| ldapBackupPort                |                                                                                                                   |
| ldapBase                      | dc=****************************                                                                                   |
| ldapBaseGroups                | ou=groups,dc=****************************                                                                         |
| ldapBaseUsers                 | ou=users,dc=*****************************                                                                         |
| ldapCacheTTL                  | 600                                                                                                               |
| ldapConfigurationActive       | 1                                                                                                                 |
| ldapDefaultPPolicyDN          |                                                                                                                   |
| ldapDynamicGroupMemberURL     |                                                                                                                   |
| ldapEmailAttribute            |                                                                                                                   |
| ldapExperiencedAdmin          | 0                                                                                                                 |
| ldapExpertUUIDGroupAttr       |                                                                                                                   |
| ldapExpertUUIDUserAttr        |                                                                                                                   |
| ldapExpertUsernameAttr        | uid                                                                                                               |
| ldapExtStorageHomeAttribute   |                                                                                                                   |
| ldapGidNumber                 | gidNumber                                                                                                         |
| ldapGroupDisplayName          | cn                                                                                                                |
| ldapGroupFilter               | (&(|(objectclass=groupOfNames))(|(cn=corneliaCloud)))                                                             |
| ldapGroupFilterGroups         | corneliaCloud                                                                                                     |
| ldapGroupFilterMode           | 0                                                                                                                 |
| ldapGroupFilterObjectclass    | groupOfNames                                                                                                      |
| ldapGroupMemberAssocAttr      | member                                                                                                            |
| ldapHost                      | wi*****************                                                                                               |
| ldapIgnoreNamingRules         |                                                                                                                   |
| ldapLoginFilter               | (&(&(|(objectclass=inetUser))(|(memberof=cn=corneliaCloud,ou=groups,dc=****************************)))(uid=%uid)) |
| ldapLoginFilterAttributes     |                                                                                                                   |
| ldapLoginFilterEmail          | 0                                                                                                                 |
| ldapLoginFilterMode           | 0                                                                                                                 |
| ldapLoginFilterUsername       | 1                                                                                                                 |
| ldapNestedGroups              | 0                                                                                                                 |
| ldapOverrideMainServer        |                                                                                                                   |
| ldapPagingSize                | 500                                                                                                               |
| ldapPort                      | 389                                                                                                               |
| ldapQuotaAttribute            |                                                                                                                   |
| ldapQuotaDefault              |                                                                                                                   |
| ldapTLS                       | 0                                                                                                                 |
| ldapUserAvatarRule            | default                                                                                                           |
| ldapUserDisplayName           | uid                                                                                                               |
| ldapUserDisplayName2          |                                                                                                                   |
| ldapUserFilter                | (&(|(objectclass=inetUser))(|(memberof=cn=corneliaCloud,ou=groups,dc=****************************)))              |
| ldapUserFilterGroups          | corneliaCloud                                                                                                     |
| ldapUserFilterMode            | 0                                                                                                                 |
| ldapUserFilterObjectclass     | inetUser                                                                                                          |
| ldapUuidGroupAttribute        | auto                                                                                                              |
| ldapUuidUserAttribute         | auto                                                                                                              |
| turnOffCertCheck              | 0                                                                                                                 |
| turnOnPasswordChange          | 0                                                                                                                 |
| useMemberOfToDetectMembership | 1                                                                                                                 |
+-------------------------------+-------------------------------------------------------------------------------------------------------------------+

```

**Comment by kesselb** [comment]
Thanks 👍

Closing as duplicate of https://github.com/nextcloud/server/issues/21250.
